counterpartylib/protocol_changes.json

```diff
@@ -110,5 +110,12 @@
         "minimum_version_revision": 0,
         "block_index": 441563,
         "testnet_block_index": 0
+    },
+    "subassets": {
+        "minimum_version_major": 9,
+        "minimum_version_minor": 55,
+        "minimum_version_revision": 2,
+        "block_index": 455000,
+        "testnet_block_index": 1088080
```

Comment:
yeah, I'd wait and do that right before release.

also, can you add to the changelog for this new version?